### ISSUE TITLE
Update snorkel to v0.5

### ISSUE THREAD

**Issue by dhimmel** [greenelab/snorkeling#5]
The big update @ajratner mentioned is now merged into master.

@danich1, let's update the snorkel submodule to HazyResearch/snorkel@36e8bcb69227e3903a8574eee775af784a53e635 (approx v0.5).

**Comment by dhimmel** [comment]
@danich1, protip -- in your commit message, you can write any of the following:

```
Closes https://github.com/greenelab/snorkeling/issues/5
Closes greenelab/snorkeling#5
Closes #5
```

Which will automatically close this issue once the commit is merged into master.
